api/proto/banyandb/database/v1/schema.proto

```diff
@@ -44,10 +44,11 @@ message Duration {
 }
 
 enum FieldType {
-    FIELD_TYPE_STRING = 0;
-    FIELD_TYPE_INT = 1;
-    FIELD_TYPE_STRING_ARRAY = 2;
-    FIELD_TYPE_INT_ARRAY = 3;
+    FIELD_TYPE_UNSPECIFIED=0;
```

Comment:
This would break the client side. Notice, only add new index is healthy. So, please update the client side proto, and rebuild.

Reply:
will pr the main repo soon

Reply:
https://github.com/apache/skywalking/pull/7462